docs/user/gcp/install_upi.md

```diff
@@ -185,13 +208,21 @@ gcloud deployment-manager deployments create ${INFRA_ID}-vpc --config 01_vpc.yam
 ```
 
 ## Configure VPC variables
+Configure the variables based on the VPC created with `01_vpc.yaml`.
+If you are using a pre-existing VPC, such as a Shared VPC, set these to the .selfLink of the targeted resources.
```

Comment:
This is not clear. i think gcloud command to show what it will be should be useful.

Reply:
Do the exports that follow not serve as an example?